examples/js/loaders/GLTFLoader.js

```diff
@@ -27,6 +27,26 @@ THREE.GLTFLoader = ( function () {
 
 			var path = this.path !== undefined ? this.path : THREE.LoaderUtils.extractUrlBase( url );
 
+			var itemGroup = '[group: ' + url + ']';
+
+			scope.manager.itemStart( itemGroup );
+
+			var _onError = function ( e ) {
+
+				if ( onError ) {
+
+					onError( e );
+
+				} else {
+
+					console.error( e );
+
+				}
+
+				scope.manager.itemError( itemGroup );
```

Comment:
I'm not 100% sure about this but i think you have to add this line:
```js
scope.manager.itemEnd( itemGroup );
```
Otherwise the code only triggers the error callback and does not raise `LoadingManagers`'s internal request counter (`itemsLoaded`).

Reply:
Example code in `FileLoader`:

https://github.com/mrdoob/three.js/blob/b293519bfb5939841dbd424ec78b1972826e21f6/src/loaders/FileLoader.js#L237-L238

Reply:
Done, thanks. 👍